### DISCUSSION TITLE
Welcome to FoE Helper Discussions!

### DISCUSSION THREAD

**Discussion by mainIine** [mainIine/foe-helfer-extension#2616] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dogwithem** [comment]
The api token is blurred. How can I fix it ?

**Comment by utezieschang-hash** [comment]
Warum funktioniert der Foe Helfer nicht mehr im Spiel obwohl ich angemeldet bin
